src/components/MiroshnikGK/Color/hextocmyk.js

```diff
@@ -0,0 +1,47 @@
+function hexToCMYK(color) {
+  let computedC = 0;
+  let computedM = 0;
+  let computedY = 0;
+  let computedK = 0;
+
+  if (color.charAt(0) !== '#') {
+    return '';
```

Comment:
Мне кажется, логичней вернуть undefined
```suggestion
    return undefined;
```

Reply:
Во всех случаях когда формат входных данных не тот.